src/pages/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
+import Achievements from './components/Achievements'
 
 const IndexPage = () => <h1>Hello world!</h1>
-
-export default IndexPage
+const Achievement = ()=> <Achievements />
+export default Achievement 
```

Comment:
Add Achivements tag inside IndexPage and export it by default

Reply:
Okay